src/index.mjs

```diff
@@ -56,11 +63,66 @@ if (!dataStoreExists) {
 await checkFeatureTypes();
 
 logger.info({ msg: `Env ready: Completed Geoserver initialization` });
-printAvocado();
-setInterval(() => { printAvocado() }, 10000000);
 
+//listen to nfs changes
+if (await isDataDirExists()) {
+  const watcher = chokidar.watch(DATASTORE_PATH, {
+    persistent: true,
+    ignoreInitial: true,
+    usePolling: true,       // use polling to detect changes - optimized for NFS
+    interval: POLLING_INTERVAL,         // how often to poll (in ms)
+    binaryInterval: POLLING_INTERVAL,
```

Comment:
what the meaning of 'binaryInterval'? is it necessary? we already got the 'interval' key

Reply:
ok, please change to POLLING_INTERVAL_MS